### ISSUE TITLE
cascade_invocations false positive with null aware operator

### ISSUE THREAD

**Issue by alexeieleusis** [dart-lang/linter#349]
```
    var foo = bar();
    // ignore: cascade_invocations
    foo?.baz();
```
